frontend/src/pages/MeetingRoom/MeetingRoom.spec.tsx

```diff
@@ -196,6 +196,19 @@ describe('MeetingRoom', () => {
     expect(screen.getByTestId('smallViewportHeader')).not.toBeNull();
   });
 
+  it('renders the captions box if it is on small tab or device and captions are enabled', () => {
+    (mui.useMediaQuery as Mock).mockReturnValue(true);
+    sessionContext.isSessionCaptioningEnabled = true;
+    render(<MeetingRoomWithProviders />);
+    expect(screen.getByTestId('captions-box')).toBeInTheDocument();
+  });
+
+  it('does not render the captions box if it is on small tab or device and captions are not enabled', () => {
+    (mui.useMediaQuery as Mock).mockReturnValue(true);
+    render(<MeetingRoomWithProviders />);
+    expect(screen.queryByTestId('captions-box')).not.toBeInTheDocument();
+  });
+
```

Comment:
Kinda goes with another comment. Can we add a case: small device and captions enabled?

Reply:
it's covered by [this unit test, right?](https://github.com/Vonage/vonage-video-react-app/pull/164/files#diff-8c5b34c816df359dac585d90500486d882a6fd077403c9ac0ee94fb125f37cfdR199-R204). or am I misunderstanding the request here? 🙏

Reply:
To be more specific, do we want this to be rendered when the user hasn't enabled captions locally? I think we want a test to confirm that.